MAUI/Segmented-Control/layout.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: post
+title: Layout in .NET MAUI Segmented control | Syncfusion
+description: Learn about the Layout support Syncfusion .NET MAUI Segmented control (SfSegmentedControl).
+platform: .NET MAUI
+control: Segmented (SfSegmented) control
+documentation: ug
+---
+ 
+# Layout in .NET MAUI Segmented control (SfSegmentedControl)
+
+## Customizing the segment width
```

Comment:
Change the segment width

Change the segment width for segmented control
Change the each segment item width

Reply:
content changed